etc/adminhtml/system.xml

```diff
@@ -591,6 +591,30 @@
                     <validate>required-entry</validate>
                 </field>
 
+                <field id="filters_and_sort_icon_position" translate="label" type="select" sortOrder="3" showInDefault="0" showInWebsite="0" showInStore="1">
+                    <label>Filters and Sort Icon Position in Mobile View</label>
+                    <source_model>Wizzy\Search\Model\Admin\Source\FiltersAndSortIconPosition</source_model>
+                    <comment>
+                        <![CDATA[
+                           By default Wizzy display Filters and Sort button at the bottom in the mobile view. Change it to top if you would like to have it changed with an offset.
+                        ]]>
+                    </comment>
+                    <validate>required-entry</validate>
+                </field>
+
+                <field id="filters_and_sort_icon_offset" translate="label" type="text" sortOrder="4" showInDefault="0" showInWebsite="0" showInStore="1">
```

Comment:
This is not a required entry. Make sure we only accept int values in this field.